### ISSUE TITLE
How to use the detectron2 model that trained by my self as pretrain model?

### ISSUE THREAD

**Issue by yh673025667** [facebookresearch/detectron2#1537]
I have trained a model with my own data, the last checkpoint is model_final.pth. But when i use this model_final.pth to pretrain my another task( set resume=False), the iteration is start from 0, while the lr is 0.0001, but not the base lr(0.01) i have set. 

It seems that the lr is initialized from the checkpoint but not the value i have set. 

Is this a bug, or just i doing something wrong?

**Comment by ppwwyyxx** [comment]
You probably need to remove "scheduler" from your checkpoint - it stores information about the current learning rate.

**Comment by yh673025667** [comment]
@ppwwyyxx Yeah, i remove 'scheduler' and it works now, thanks

**Comment by indigoviolet** [comment]
```
def clean_model_for_resume(pth: Path, output_pth: Path):
    m = torch.load(str(pth))
    del m['scheduler']
    del m['iteration']
    torch.save(m, output_pth)
```
something like this should work (@pvtien96 )

**Comment by pvtien96** [comment]
@indigoviolet thanks, I'll try and feed back.

**Comment by yh673025667** [comment]
@pvtien96 Sorry to respond late, you can try the following code.
`
model = torch.load(your_model_pth)
del model['optimizer']
del model['scheduler']
del model['iteration']

torch.save(model, save_path)
`
